packages/core/logging/core-logging-browser-internal/src/layouts/pattern_layout.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  PatternLayout as BasePatternLayout,
+  type Conversion,
+  LoggerConversion,
+  LevelConversion,
+  MetaConversion,
+  MessageConversion,
+  DateConversion,
+} from '@kbn/core-logging-common-internal';
+
+const DEFAULT_PATTERN = `[%date][%level][%logger] %message`;
+
+const conversions: Conversion[] = [
+  LoggerConversion,
+  MessageConversion,
+  LevelConversion,
+  MetaConversion,
+  DateConversion,
+];
+
+/**
+ * Layout that formats `LogRecord` using the `pattern` string with optional
+ * color highlighting (eg. to make log messages easier to read in the terminal).
+ * @internal
+ */
+export class PatternLayout extends BasePatternLayout {
```

Comment:
Inheriting the base `PatternLayout`. Only the constructor is overridden to manually pass some parameters to it:
- `highlight: false` given it's not supported on the browser
- the browser-side list of supported `conversions` (excluding the `pid` one)

Reply:
Is there a reason for renaming the base pattern layout [here](https://github.com/elastic/kibana/pull/144107/files/75b796a8d4894669ce44d77fd3c2e870f37a398a#diff-cf2065f1bcf0005b098de3067e688b904bc4b5224c5734527e8913343918c254R10) rather than in the common code?